docs/guides/add-onboarding-flow.mdx

```diff
@@ -100,12 +99,12 @@ export default authMiddleware({
     }
 
     // If the user is logged in and the route is protected, let them view.
-    if (userId && !auth.isPublicRoute) return NextResponse.next();
+    if (userId && !isPublicRoute(req)) return NextResponse.next();
 
     // If the route is public, anyone can view it.
-    if (auth.isPublicRoute) return NextResponse.next();
-  },
-});
+    if (isPublicRoute(req)) return NextResponse.next();
+  }
+);
```

Comment:
This should be removed. Routes are public by default, so if a restriction is not matched then it will public.